internal/k8s/metrics/node/node.go

```diff
@@ -61,7 +61,7 @@ func New(opts ...Option) NodeWatcher {
 }
 
 func (r *reconciler) Reconcile(req reconcile.Request) (res reconcile.Result, err error) {
-	m := &metrics.NodeMetricsList{}
+	m := new(metrics.NodeMetricsList)
```

Comment:
It is not essential, but I think this fix and the similar ones should be in the other PR.
How about it?

Reply:
I think so, too.:bow:
But it's not a big change, so I don't think this change is a problem.
And kpango asked us to include this change in the same PR.